src/classes/issues-processor.ts

```diff
@@ -268,6 +317,130 @@ export class IssuesProcessor {
       );
     }
 
+    if (this.options.mlflow) {
+      const {startDate, daysBeforeReplyReminder, daysBeforeAssigneeReminder} =
+        this.options;
+      if (
+        issue.milestone ||
+        issue.isStale ||
+        (startDate &&
+          !isDateMoreRecentThan(
+            new Date(issue.created_at),
+            new Date(startDate)
+          ))
+      ) {
+        return;
+      }
+
+      const reminderToMaintainers = 'Reminder to MLflow maintainers';
+      if (issue.isPullRequest) {
+        // TODO
+      } else {
+        const hasClosingPr = issue.labels.some(
+          ({name}) => name === 'has-closing-pr'
+        );
+        if (hasClosingPr) {
+          issueLogger.info('This issue has a closing PR');
+          return;
+        }
+        const comments = await this.listIssueComments(issue, issue.created_at);
+        issueLogger.info(
+          `The number of comments in this issue: ${comments.length}`
+        );
+
+        issueLogger.info(
+          `Assignees on this issue: ${issue.assignees.map(({login}) => login)}`
+        );
+
+        if (comments.length > 0) {
+          const lastComment = comments[comments.length - 1];
+          issueLogger.info(
+            `The last comment was posted by ${lastComment.user?.login}`
+          );
+          const isBotComment = lastComment.user?.type !== 'User';
+          issueLogger.info(`Did a bot post the last comment? ${isBotComment}`);
+
+          const lastCommentCreatedAt =
+            lastComment.created_at || IssuesProcessor._getNow();
+          const daysSinceCreated =
+            IssuesProcessor._getDaysSince(lastCommentCreatedAt);
+          issueLogger.info(
+            `Days since the last comment was posted: ${daysSinceCreated.toFixed(
+              2
+            )}`
+          );
+          if (
+            !isBotComment &&
+            !IssuesProcessor._updatedSince(
+              lastCommentCreatedAt,
+              daysBeforeReplyReminder
+            )
+          ) {
+            const lastCommentPostedByMaintainer = lastComment.user
+              ? this.isPostedByMaintainer(lastComment.user.login)
+              : false;
+            issueLogger.info(
+              `Did a maintainer post the last comment? ${lastCommentPostedByMaintainer}`
+            );
+            if (lastCommentPostedByMaintainer) {
+              const mention = issue.user ? `@${issue.user.login}` : '';
+              await this.createComment(issue, `${mention} Any updates here?`);
+              return;
+            } else {
+              await this.createComment(
+                issue,
+                `${reminderToMaintainers}. Please reply to comments.`
```

Comment:
Can we tag the people who are assigned to the issue in the comment?

Reply:
Sure!